docs/dvl/gui/dashboard.md

```diff
@@ -9,6 +9,34 @@ The default page is a dashboard which provides a summary and visualisation of bo
 
 ![](../../img/dvl_gui_dashboard.png)
 
+## Dashboard Parameters
+
+### Doppler Velocity
+Parameter                               |Description
+----------------------------------------|---------------------------------------------------------------------
+Speed                                   | Combined speed from x, y, z velocities.
+Altitude                                | Distance from seabed to DVL (unless beams blocked by something else).
+Figure of merit(See note below)         | Maximum uncertainty of measurement given in m/s.
+Ping rate                               | Rate at which data velocity is being outputted (depends on velocity).
+Velocity valid                          | Indicates if the velocity is valid to be used.
+
+!!! Note
+	Figure of merit (FOM) represent the maximum possible uncertainty represents the largest potential error in the measured speed from our DVL. This value is derived from the covariance matrix, which identifies the direction with the highest uncertainty in the sensor data. It provides a worst-case error margin. The maximum amount by which the actual speed may differ from the measured speed due to various factors from environment such as acoustic noise from other sources.
+
+    Users can use this figure of merit to assess the reliability of the speed measurements and determine whether the level of uncertainty meets their application requirements. This value allows for better decision-making and operational planning, ensuring that even in the most challenging conditions, the DVL operates within predictable error limits. 
```

Comment:
Is this intended to be part of the "note"? If so the spacing is inconsistent here and the previous paragraph, so it will probably not look right (assumed, not looked at the rendered result)

Reply:
[c47ee3a](https://github.com/waterlinked/waterlinked.github.io/pull/30/commits/c47ee3a15bd9fb04af82693ca35e14b800fa98b1)